common/djangoapps/student/migrations/0012_userprofile_specialty.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
One migration per PR.

Reply:
thank you - I'll keep that in mind in the future. 
Because I will be now adding 'profession' and 'specialty' fields to the UserProfile.meta field there won't need to be any migrations once I make that change